databricks/koalas/indexing.py

```diff
@@ -75,12 +75,169 @@ def _unfold(key, col):
     return rows_sel, cols_sel
 
 
-class SparkDataFrameLocator(object):
+class LocIndexer(object):
     """
-    A locator to slice a group of rows and columns by conditional and label(s).
+    Access a group of rows and columns by label(s) or a boolean Series.
 
-    Allowed inputs are a slice with all indices or conditional for rows, and string(s) or
-    :class:`Column`(s) for cols.
+    ``.loc[]`` is primarily label based, but may also be used with a
+    conditional boolean Series derived from the DataFrame or Series.
+
+    Allowed inputs are:
+
+    - A single label, e.g. ``5`` or ``'a'``, (note that ``5`` is
+      interpreted as a *label* of the index, and **never** as an
+      integer position along the index) for column selection.
+
+      .. warning:: A single label for row selection is not allowed
```

Comment:
warning looks really scary. when i open the doc page for this, it's flooded with red boxes. maybe just change it to a note or even just an inline sentence.

![Screen Shot 2019-05-14 at 12 28 30 AM](https://user-images.githubusercontent.com/323388/57679035-435f8100-75df-11e9-9f24-fca495afccb6.png)

Reply:
sure. will make it note.

Reply:
take a look at the generated doc to make sure it's not too daunting.